docs/source/examples/check_env.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+
+from conu import DockerBackend, DockerRunBuilder
+
+image = DockerBackend.ImageClass('fedora')
+cmd = DockerRunBuilder(additional_opts=['-i', '-e', 'KEY=space'])
+
+cont = image.run_via_binary_in_foreground(cmd, popen_params={"stdin": subprocess.PIPE})
```

Comment:
I'm not sure if we have the stdin trick in our docs. It should definitely be mentioned in docstring of `run_via_binary_in_foreground`.